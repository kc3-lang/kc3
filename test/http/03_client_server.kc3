quote server = Socket.listen("localhost", "58000")
server = Socket.listen("localhost", "58000")
quote client = Socket.Buf.connect("localhost", "58000")
client = Socket.Buf.connect("localhost", "58000")
quote server_client = Socket.Buf.accept(server)
server_client = Socket.Buf.accept(server)
quote Socket.Buf.close(client)
Socket.Buf.close(client)
quote Socket.Buf.close(server_client)
Socket.Buf.close(server_client)
quote Socket.close(server)
Socket.close(server)
