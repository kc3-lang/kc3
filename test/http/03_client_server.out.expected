require Socket
true
require Socket.Buf
true
(server = Socket.listen("127.0.0.1", "57000")) ; void
void
(client = Socket.Buf.connect("127.0.0.1", "57000")) ; void
void
(server_client = Socket.Buf.accept(server)) ; void
void
Socket.Buf.close(server_client)
void
Socket.Buf.close(client)
void
Socket.close(server)
void
